Composer/packages/client/scripts/build.js

```diff
@@ -35,6 +35,16 @@ const WARN_AFTER_CHUNK_GZIP_SIZE = 1024 * 1024;
 
 const isInteractive = process.stdout.isTTY;
 
+//copy monaco editor resouce to public folder to support offline usage
+const monacoEditor = path.resolve(__dirname, '../../../node_modules/monaco-editor/min');
+const monacoEditorInPublicFolder = path.resolve(__dirname, '../public/min');
+fs.copySync(monacoEditor, monacoEditorInPublicFolder);
+
+//copy fabric fonts to public folder to support offline usage
+const fabricFonts = path.resolve(__dirname, '../../../node_modules/@uifabric/icons/fonts');
+const fabricFontsInPublicFolder = path.resolve(__dirname, '../public/fonts');
+fs.copySync(fabricFonts, fabricFontsInPublicFolder);
+
```

Comment:
Doing this as part of the prod build only seems like it would break local development. Is that true? I was thinking it should happen as a postinstall script.